src/Phinx/Db/Adapter/PdoAdapter.php

```diff
@@ -406,8 +406,18 @@ public function getVersions()
     public function getVersionLog()
     {
         $result = array();
-        $rows = $this->fetchAll(sprintf('SELECT * FROM %s ORDER BY version ASC', $this->getSchemaTableName()));
-        foreach ($rows as $version) {
+
+        switch ($this->options['version_order']) {
+            case \Phinx\Config\Config::VERSION_ORDER_CREATION_TIME:
```

Comment:
Move the default option to the bottom of the list and add a `default:` clause, so that if nothing is setup right, the default option will operate appropriately.